Loading/ru.lproj/InfoPlist.strings

```diff
@@ -1 +1 @@
-CFBundleName = "Загрузка";
CFBundleDisplayName = "Загрузка";
NSHumanReadableCopyright = "© 2015 Mike McFadden / Bonzai Программы\nВсе права защищены.";
\ No newline at end of file
+CFBundleName = "Loading";
CFBundleDisplayName = "Loading";
NSHumanReadableCopyright = "© 2015 Mike McFadden / Bonzai Apps\nВсе права защищены.";
```

Comment:
Well, I can assure you that application names in English are totally fine for Russian OS X users. Same goes for company names.

I'll translate the «app» thing right now…